test/core/test/exports.test.ts

```diff
@@ -0,0 +1,205 @@
+import { resolve } from 'node:path'
+import { expect, it } from 'vitest'
+import { getPackageExportsManifest } from 'vitest-package-exports'
+
+it('exports snapshot', async () => {
+  const manifest = await getPackageExportsManifest({
+    importMode: 'package', // or 'dist' or 'package'
+    cwd: resolve(import.meta.dirname, '../../../packages/vitest'),
+  })
+
+  expect(manifest.exports)
+    .toMatchInlineSnapshot(`
+      {
+        ".": {
+          "afterAll": "function",
+          "afterEach": "function",
+          "assert": "function",
+          "assertType": "function",
+          "beforeAll": "function",
+          "beforeEach": "function",
+          "bench": "function",
+          "chai": "object",
+          "createExpect": "function",
+          "describe": "function",
+          "expect": "function",
+          "expectTypeOf": "function",
+          "getRunningMode": "function",
+          "inject": "function",
+          "isFirstRun": "function",
+          "isWatchMode": "function",
+          "it": "function",
+          "onTestFailed": "function",
+          "onTestFinished": "function",
+          "runOnce": "function",
+          "should": "function",
+          "suite": "function",
+          "test": "function",
+          "vi": "object",
+          "vitest": "object",
+        },
+        "./browser": {
+          "SpyModule": "object",
+          "collectTests": "function",
+          "getCoverageProvider": "function",
```

Comment:
This shouldn't be exposed. It's only used on main thread / node side.

Reply:
Seems like this also leads to types leaking to `@vitest/browser` in local dev (https://github.com/vitest-dev/vitest/pull/7619)

Reply:
let's remove it